util/auth.js

```diff
@@ -0,0 +1,11 @@
+const jwt = require('jsonwebtoken');
+const env = require('../env');
+
+const authorizedOnly = resolver => (root, args, context) => {
+  const { token } = context.headers;
+  if (jwt.verufy(token, env.secret)) {
+    return resolver(root, args, context);
+  } throw new Error('Frobidden');
```

Comment:
Put this throw to the next line. Also, maybe it would be better to use our `APIError` class?